site/_docs/installation.md

```diff
@@ -31,16 +32,18 @@ earlier, for CoffeeScript support).
   </p>
 </div>
 
-## Install with RubyGems
+## Install with Bundler and RubyGems
 
-The best way to install Jekyll is via
+The best way to install Jekyll is using a Gemfile via Bundler
 [RubyGems](http://rubygems.org/pages/download). At the terminal prompt,
 simply run the following command to install Jekyll:
 
 {% highlight shell %}
-$ gem install jekyll
+$ gem install bundler
 {% endhighlight %}
 
+Now you can install Jekyll simply by running `bundle install`.
```

Comment:
It was in the original PR (#4368) but later removed. If `new.rb` defines `gemfile_contents` and creates a scaffold, that's one less step we need to tell people about.

<img width="581" alt="831b9a92-c0f5-11e5-8f9c-8ca6b686c438" src="https://cloud.githubusercontent.com/assets/1534882/17463757/2a839c12-5c9b-11e6-9513-030609fbdb7c.png">


@ashmaroli has done a fair bit of work on this in #5203 to extend the capabilities of `new` and flesh out the site template.

Might also be useful for themes 🤔